hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DatanodeDeletedBlockTransactions.java

```diff
@@ -18,8 +18,10 @@
 
 import org.apache.hadoop.hdds.protocol.proto
     .StorageContainerDatanodeProtocolProtos.DeletedBlocksTransaction;
+import org.apache.hadoop.hdfs.protocol.datatransfer.Op;
```

Comment:
This is an import only file change, please remove it.

Reply:
Sure, thank you for the reminder.